package.json

```diff
@@ -47,7 +47,7 @@
     "moment-timezone": "^0.5.11",
     "node-sass": "^4.4.0",
     "object-unfreeze": "^1.1.0",
-    "pathable-scripts": "git+https://github.com/pathable/pathable-scripts.git",
+    "pathable-scripts": "https://github.com/pathable/pathable-scripts.git",
```

Comment:
It should be only in `devDependencies`. Probably it was included automatically by dime's env at `dependencies`. Remember to change the protocol to `git+https`.